src/client/Client.cpp

```diff
@@ -185,6 +209,22 @@ void Client::SetServerDropRate() {
       boost::bind(&Client::HandleWriteResult, this, _1));
 }
 
+void Client::KillServersRandomly(int max_failures) {
+  auto request = message::KillServer();
+  auto encoded = message::Message(request.Encode(),
+      message::MessageType::kKillServer).Encode();
+  unsigned int seed = 0;
+
+  for (int i = 0; i < max_failures; i++) {
+    int sleep_ms = rand_r(&seed) % kMaxTimeBetweenKill + 1;
+    boost::this_thread::sleep_for(boost::chrono::milliseconds(sleep_ms));
```

Comment:
Use std::chrono here instead of boost::chrono. See elsewhere in file for examples. Reduces number of dependencies needed.